apps/mp/services.py

```diff
@@ -562,24 +563,53 @@ def parse_dependency(model, dependency):
             return
         try:
             field = getattr(model, attr_name)
-            # for ReverseSingleRelatedObjectDescriptor
-            if hasattr(field, 'field'):
-                is_reverse = field.__class__.__name__.startswith('Reverse')
-                field = field.field
-                model = field.rel.related_model if is_reverse else field.rel.to
-                related_name = (
-                    field.name if is_reverse else field.rel.related_name
-                )
+            relation_class_name = field.__class__.__name__
+            # django.VERSION is a tuple. First item is major version
+            if django.VERSION[0] == 1:
+                if hasattr(field, 'field'):
+                    # for ReverseSingleRelatedObjectDescriptor
+                    is_reverse = relation_class_name.startswith('Reverse')
+                    field = field.field
+                    model = field.rel.related_model if is_reverse else field.rel.to
+                    related_name = (
+                        field.name if is_reverse else field.rel.related_name
+                    )
 
-                if not related_name or related_name == '+':
-                    raise Exception(
-                        "Fields with no explicit related name aren't supported for now: {}".format(
-                            field
-                        )
+                else:
+                    model = field.related.model
+                    related_name = field.related.field.name
+            else:  # Django version is >= 2
```

Comment:
What versions of Django has this been tested with? Can we leave some kind of comment indicating the level of confidence for this piece of code being compatible with Django 2, and Django >= 3?